package day_one

func Calc() (int, int) {
	// return - this line cause error
	return 1, 2
}
